libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -2605,44 +2618,28 @@ bool AudioIoCallback::FillOutputBuffers(
 
    bool drop = false;        // Sequence should become silent.
    bool discardable = false; // Sequence has already been faded to silence.
+   // mPlaybackBuffers buffers correspond many-to-one with mPlaybackSequences
    size_t iBuffer = 0;
-   for (unsigned t = 0; t < numPlaybackSequences; t++)
-   {
-      auto vt = mPlaybackSequences[t].get();
+   for (unsigned tt = 0; tt < numPlaybackSequences; ++tt) {
+      auto vt = mPlaybackSequences[tt].get();
       const auto width = vt->NChannels();
```

Comment:
You've called this `nChannels` elsewhere, better stay consistent to ease reading.